zrml/prediction-markets/src/lib.rs

```diff
@@ -2016,8 +2148,10 @@ mod pallet {
             Ok(())
         }
 
-        pub(crate) fn market_close_manager<F>(
-            now: T::BlockNumber,
+        pub(crate) fn market_open_manager<F>(
+            block_number: T::BlockNumber,
+            last_time_frame: TimeFrame,
+            current_time_frame: TimeFrame,
```

Comment:
The `market_close_manager` and `market_open_manager` only differ in the storage they use. I propose to create a common functions that takes the storage as an additional parameter and then use that instead, effectively deduplicating the code.

Reply:
Yeah, great suggestion. I'll get on that :thumbsup:

Reply:
Mitigated in b38035681aa03a3fc9533fb1438a450739879744